src/main/java/seedu/easylog/commands/itemscommands/ItemsPromptPriceCommand.java

```diff
@@ -18,9 +17,10 @@
     public BigDecimal execute() throws NullItemPriceException, EmptyItemPriceException, InvalidItemPriceException,
             NonNumericItemPriceException {
         ui.promptItemPrice();
+
         String priceInString = ui.askForUserInput();
         try {
-            Integer.parseInt(priceInString);
+            Double.parseDouble(priceInString);
         } catch (NumberFormatException e) {
             throw new NonNumericItemPriceException();
         }
```

Comment:
think this part is irrelevant, but we can leave it for now

Reply:
numberformat exception can be caught below in the processPriceInput

Reply:
I will check all exceptions and make sure they are caught at right place and print correct error message when adding an item later today :)